setup.py

```diff
@@ -62,13 +62,13 @@
     'Flask-Login>=0.3.2,<0.5.0',
     'Flask-WTF>=0.14.3',
     'invenio-base>=1.2.2',
+    'invenio-accounts>=1.2.0',
```

Comment:
minor: 1.2.0 is a broken build, maybe 1.2.1 is a better candidate, although functionality wise, might not change anything.
```suggestion
    'invenio-accounts>=1.2.1',
```